frontend/src/components/project/Buttons/ContactCreatorButton.js

```diff
@@ -99,85 +95,58 @@ export default function ContactCreatorButton({
     : texts.responsible_person_project;
   const buttonText = texts.contact;
 
-  if (small) {
-    return (
-      <Button
-        variant="contained"
-        color="primary"
-        startIcon={<SendIcon />}
-        endIcon={<Avatar src={creatorImageURL} className={classes.smallAvatar} />}
-        onClick={handleClickContact}
-        className={className}
-      >
-        {buttonText}
-      </Button>
-    );
-  } else if (tiny) {
-    return (
-      <Button
-        variant="contained"
-        color="primary"
-        onClick={handleClickContact}
-        className={className}
-      >
-        {buttonText}
-      </Button>
-    );
-  } else if (large) {
-    return (
-      <div className={`${classes.largeButton} ${className}`} onClick={handleClickContact}>
-        <DetailledContactCreatorInfo
-          creatorName={creatorName}
-          creatorImageURL={creatorImageURL}
-          creatorsRoleInProject={creatorsRoleInProject}
-        />
-        <Button variant="contained" color="primary">
-          {buttonText}
-        </Button>
-      </div>
-    );
-  } else {
-    return (
+  return (
+    <div
+      className={withInfoCard ? `${classes.root} ${className}` : null}
+      onMouseEnter={handleMouseEnter}
+      onMouseLeave={handleMouseLeave}
+      onClick={handleClickContact}
+    >
       <div
-        className={`${classes.root} ${className}`}
-        onMouseEnter={handleMouseEnter}
-        onMouseLeave={handleMouseLeave}
-        onClick={handleClickContact}
+        className={
```

Comment:
Yes, it's what I meant :-). I would maybe simplify this:

```
collapsable  ? `${classes.collapsableContainer} ${classes.applyCustomCardWidth}` : classes.applyCustomCardWidth
```
using:
```
`${collapse ? classes.collapsableContainer : ''} ${classes.applyCustomCardWidth}`
```
but that's only my preference, you can keep it as it is. :-)

Reply:
It looks cleaner the way you suggested, I changed it to that :)